airflow/hooks/dbapi_hook.py

```diff
@@ -171,7 +171,7 @@ def run(self, sql, autocommit=False, parameters=None):
 
             # If autocommit was set to False for db that supports autocommit,
             # or if db does not supports autocommit, we do a manual commit.
-            if not getattr(conn, 'autocommit', False):
+            if not self.get_autocommit(conn):
```

Comment:
I think it will still not commit when autocommit set to True. Remove the `"not"` ?

Reply:
We should not commit when autocommit is set the `True`. That was the root cause--each cursor.execute() will commit the query if autocommit is set to `True` and we only do manual commit if autocommit is not set.

Reply:
All right. LGTM 👍